GitUI/UserControls/RevisionGrid/Graph/RevisionGraph.cs

```diff
@@ -180,7 +180,7 @@ public void HighlightBranch(ObjectId id)
         /// <summary>
         /// Add a single revision from the git log.
         /// </summary>
-        public void Add(GitRevision revision, RevisionNodeFlags types)
+        public void Add(GitRevision revision, RevisionNodeFlags types, bool addParentGraph = true)
```

Comment:
Please add xml-doc explaining the new argument.

Reply:
Will update